python/paddle/hapi/hub.py

```diff
@@ -0,0 +1,260 @@
+            raise RuntimeError('Missing dependencies: {}'.format(', '.join(
+                missing_deps)))
+
+
+def list(repo_dir, source='github', force_reload=False):
+    r"""
+    List all entrypoints available in `github` hubconf.
+
+    Args:
+        repo_dir(str): github or local path
+            github path (str): a str with format "repo_owner/repo_name[:tag_name]" with an optional
+                tag/branch. The default branch is `master` if not specified.
+            local path (str): local repo path
+        source (str): `github` | `local`
+        force_reload (bool, optional): whether to discard the existing cache and force a fresh download, default is `False`.
+    Returns:
+        entrypoints: a list of available entrypoint names
+
+    Example:
+        ```python
+        import paddle
+
+        paddle.hub.help('lyuwenyu/PaddleClas:hub_L', source='github', force_reload=True)
+
+        ```
+    """
+    if source not in ('github', 'local'):
+        raise ValueError(
+            'Unknown source: "{}". Allowed values: "github" | "local".'.format(
+                source))
+
+    if source == 'github':
+        repo_dir = _get_cache_or_reload(repo_dir, force_reload, True)
+
+    sys.path.insert(0, repo_dir)
+    hub_module = import_module(MODULE_HUBCONF, repo_dir + '/' + MODULE_HUBCONF)
+    sys.path.remove(repo_dir)
+
+    entrypoints = [
+        f for f in dir(hub_module)
+        if callable(getattr(hub_module, f)) and not f.startswith('_')
+    ]
+
+    return entrypoints
+
+
+def help(repo_dir, model, source='github', force_reload=False):
+    """
+    Show help information of model
+
+    Args:
+        repo_dir(str): github or local path
+            github path (str): a str with format "repo_owner/repo_name[:tag_name]" with an optional
+                tag/branch. The default branch is `master` if not specified.
+            local path (str): local repo path
+        model (str): model name
+        source (str): source of repo_dir, `github` | `local`, default is `github`
+        force_reload (bool, optional): default is `False`
+    Return:
+        docs
+
+    Example:
+        ```python
+        import paddle
+
+        paddle.hub.help('lyuwenyu/PaddleClas:hub_L', model='ResNet18Test', source='github')
+        ```
+    """
+    if source not in ('github', 'local'):
+        raise ValueError(
+            'Unknown source: "{}". Allowed values: "github" | "local".'.format(
+                source))
+
+    if source == 'github':
+        repo_dir = _get_cache_or_reload(repo_dir, force_reload, True)
+
+    sys.path.insert(0, repo_dir)
+    hub_module = import_module(MODULE_HUBCONF, repo_dir + '/' + MODULE_HUBCONF)
+    sys.path.remove(repo_dir)
+
+    entry = _load_entry_from_hubconf(hub_module, model)
+
+    return entry.__doc__
+
+
+def load(repo_dir, model, *args, source='github', force_reload=False, **kwargs):
+    """
+    Load model
+
+    Args:
+        repo_dir(str): github or local path
```

Comment:
as offline discussion, also can add gitee

Reply:
ok, will push it after testing offline